build/tutorials/tutorials-contest/red-dev-avalanche-chainlink-tutorial/chainlink-external-adaptor/app.js

```diff
@@ -0,0 +1,21 @@
+import conf from 'nconf';
```

Comment:
Port js -> ts

Reply:
do u want me to change the entire project to ts? or only the app.js? @cgcardona